stdlib/public/core/SIMDVector.swift

```diff
@@ -90,13 +90,6 @@ extension SIMD {
   
   /// A vector with the specified value in all lanes.
   @_transparent
-  // SWIFT_ENABLE_TENSORFLOW
-  @differentiable(vjp: _vjpInit(repeating:)
```

Comment:
Could you please keep `@differentiable` attributes but remove the `vjp: xxx` argument? `@differentiable` attributes on public original declarations ensures that derivatives are publicly accessible.

This has been done consistently in other `@differentiable(jvp:vjp:) -> @derivative` PRs.

Reply:
Oh okay sure!